src/__tests__/DropdownMenu.spec.js

```diff
@@ -157,7 +157,7 @@ describe('DropdownMenu', () => {
       }
     );
 
-    expect(wrapper.find(Popper).prop('modifiers')).toBe(undefined);
+    expect(wrapper.find(Popper).prop('modifiers')).toEqual({});
```

Comment:
Without the default prop setting it to and object, this should be able to be reverted